test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ExternalMavenRepoIntegrationTest.java

```diff
@@ -11,50 +11,125 @@
 import org.kie.server.api.model.instance.TaskSummary;
 import org.kie.server.client.KieServicesClient;
 import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.RuleServicesClient;
 import org.kie.server.client.UserTaskServicesClient;
+import org.kie.server.integrationtests.shared.KieServerAssert;
 
-public class ExternalMavenRepoIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerDatabaseScenario> {
+@RunWith(Parameterized.class)
+public class ExternalMavenRepoIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerScenario> {
 
-    @Override
-    protected WorkbenchKieServerDatabaseScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
-        return deploymentScenarioFactory.getWorkbenchKieServerDatabaseScenarioBuilder()
+    @Parameter
+    public WorkbenchKieServerScenario workbenchKieServerScenario;
+
+    @Parameters
+    public static Collection<Object[]> data() {
+        DeploymentScenarioBuilderFactory deploymentScenarioFactory = DeploymentScenarioBuilderFactoryLoader.getInstance();
+
+        WorkbenchKieServerScenario workbenchKieServerScenario = deploymentScenarioFactory.getWorkbenchKieServerScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .build();
+        WorkbenchKieServerDatabaseScenario workbenchKieServerDatabaseScenario = deploymentScenarioFactory.getWorkbenchKieServerDatabaseScenarioBuilder()
                 .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
                 .build();
+
+        return Arrays.asList(new Object[][]{
+            {workbenchKieServerScenario}, {workbenchKieServerDatabaseScenario}
+        });
+    }
+
+    private static final String LIST_NAME = "list";
+    private static final String LIST_OUTPUT_NAME = "output-list";
+    private static final String KIE_SESSION = "defaultKieSession";
+    private static final String FAST_RULE = "Fast rule executed";
+    private static final String SLOW_RULE = "Slow rule executed";
+
+    private static final String RULE_CONTAINER_ID = "rule-container-id";
+    private static KieCommands commandsFactory = KieServices.Factory.get().getCommands();
+
+    @Override
+    protected WorkbenchKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return workbenchKieServerScenario;
     }
 
     @Before
     public void deployMavenProject() {
         MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/definition-project-snapshot").getFile());
+        MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/rule-project").getFile());
     }
 
     @Test
     public void testProcessFromExternalMavenRepo() {
```

Comment:
Can you please split this test into 2? One for drools and one for jbpm?

Reply:
For this I'll open a new PR with categories to run jBPM or Drools only